arch/arm/boot/dts/overlays/pitft22-overlay.dts

```diff
@@ -43,7 +43,7 @@
                         status = "okay";
 
                         pitft: pitft@0{
-                                compatible = "ilitek,ili9340";
+                                compatible = "ilitek,ili9340", "adafruit,yx240qv29";
```

Comment:
Sorry, I think was probably my error in the pitft35-resistive overlay.
"ilitek,ili9340" is the fbtft compatible. "adafruit,yx240qv29" is the tinydrm one. We already have the DRM override rewriting this correctly, so the original line here is correct.